cloudinit/settings.py

```diff
@@ -78,3 +78,27 @@
 FREQUENCIES = [PER_INSTANCE, PER_ALWAYS, PER_ONCE]
 
 HOTPLUG_ENABLED_FILE = "/var/lib/cloud/hotplug.enabled"
+
+# DEPRECATION_INFO_BOUNDARY is used by distros to configure at which version to
+# start logging deprecations at a level higher than INFO.
+#
+# The default value "devel" tells cloud-init to log all deprecations higher
+# than INFO. This value may be overriden by downstreams in order to maintain
+# stable behavior across releases.
+#
+# Jsonschema key deprecations and inline logger deprecations include a
+# deprecated_version key. When the variable below is set to a version,
+# cloud-init will use that version as a demarcation point. Deprecations which
+# are added after this version will be logged as at an INFO level. Deprecations
+# which predate this version will be logged at the higher DEPRECATED level.
+# Downstreams that want stable log behavior may set the variable below to the
+# first version released in their stable distro. By doing this, they can expect
+# that newly added deprecations will be logged at INFO level.
+#
+# format:
+#
+# <value> :: = <default> | <version>
```

Comment:
```suggestion
# <value> :: = devel | <version>
```

Reply:
I guess you were expressing that default == "devel" on the next line. so no need to accept this comment

Reply:
correct, we probably don't need to express this in BNF, but it's what came to my mind in the moment.

Now that I look though the patch part is wrong, I'll fix that.